backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -146,3 +263,67 @@ def test_review_requested(self):
         self.assertEqual(entry.object_1, card.recruit_project)
         self.assertEqual(entry.object_2, None)
         self.assertEqual(entry.event_type.name, creators.CARD_REVIEW_REQUESTED)
+
+
+class log_card_review_feedback_correctly_called_Tests(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "agilecard-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
+    SUPPRESS_TEST_GET_LIST = True
+
+    @mock.patch(
+        "curriculum_tracking.activity_log_entry_creators.log_card_moved_to_review_feedback"
+    )
+    @mock.patch(
+        "curriculum_tracking.activity_log_entry_creators.log_card_moved_to_complete"
+    )
+    def test_card_review_feedback_called(
+        self, log_card_moved_to_complete, log_card_moved_to_review_feedback
+    ):
+        actor_user = UserFactory(is_superuser=True)
+        card = factories.AgileCardFactory(
+            status=AgileCard.IN_REVIEW,
+            content_item=factories.ProjectContentItemFactory(
+                project_submission_type=ContentItem.LINK, template_repo=None
+            ),
+        )
+        self.login(actor_user)
+
+        add_review_url = f"{self.get_instance_url(card.id)}add_review/"
+        response = self.client.post(
+            add_review_url, data={"status": NOT_YET_COMPETENT, "comments": "so sad"}
+        )
+        self.assertEqual(response.status_code, 200)
+
+        card.refresh_from_db()
+
+        log_card_moved_to_review_feedback.assert_called_with(card, actor_user)
+        log_card_moved_to_complete.assert_not_called()
+
+    @mock.patch(
+        "curriculum_tracking.activity_log_entry_creators.log_card_moved_to_review_feedback"
+    )
+    @mock.patch(
+        "curriculum_tracking.activity_log_entry_creators.log_card_moved_to_complete"
+    )
+    def test_card_moved_to_complete_called(
+        self, log_card_moved_to_complete, log_card_moved_to_review_feedback
+    ):
+        actor_user = UserFactory(is_superuser=True)
+        card = factories.AgileCardFactory(
+            status=AgileCard.IN_REVIEW,
+            content_item=factories.ProjectContentItemFactory(
+                project_submission_type=ContentItem.LINK, template_repo=None
+            ),
+        )
+        self.login(actor_user)
+
+        add_review_url = f"{self.get_instance_url(card.id)}add_review/"
+        response = self.client.post(
+            add_review_url, data={"status": COMPETENT, "comments": "woohoo"}
+        )
+        self.assertEqual(response.status_code, 200)
+
+        card.refresh_from_db()
+        project_review = RecruitProjectReview.objects.first()
+        log_card_moved_to_complete.assert_called_with(card, actor_user)
+        log_card_moved_to_review_feedback.assert_not_called()
```

Comment:
Same reasoning as above. Also, `project_review` isn't being used anywhere.
```suggestion
        card.refresh_from_db()
        self.assertEqual(card.status, AgileCard.COMPLETE)
        log_card_moved_to_complete.assert_called_with(card, actor_user)
        log_card_moved_to_review_feedback.assert_not_called()
```

Reply:
ah, yes, some bad pattern matching ;)